src/iop/lut3d.c

```diff
@@ -1036,10 +1133,9 @@ void gui_init(dt_iop_module_t *self)
   dt_bauhaus_widget_set_label(g->colorspace, NULL, _("application color space"));
   dt_bauhaus_combobox_add(g->colorspace, _("sRGB"));
   dt_bauhaus_combobox_add(g->colorspace, _("REC.709"));
-  dt_bauhaus_combobox_add(g->colorspace, _("lin sRGB/REC.709"));
-  dt_bauhaus_combobox_add(g->colorspace, _("lin prophoto RGB"));
+  dt_bauhaus_combobox_add(g->colorspace, _("linear REC.709"));
   gtk_box_pack_start(GTK_BOX(self->widget), GTK_WIDGET(g->colorspace) , TRUE, TRUE, 0);
-  gtk_widget_set_tooltip_text(g->colorspace, _("select the color space in which the LUT has to be applied"));
+  gtk_widget_set_tooltip_text(g->colorspace, _("select the color space in which the LUT has to be applied\n"));
```

Comment:
please no \n on tooltip. if space is needed this should be done in CSS for all tooltip anyway.

Reply:
done